spectacles.gemspec

```diff
@@ -21,13 +21,14 @@ Gem::Specification.new do |gem|
   ##
   # Dependencies
   #
-  gem.required_ruby_version = ">= 2.2.0"
-  gem.add_dependency "activerecord", ">= 3.2.0", "~> 6.1.0"
-  gem.add_dependency "activesupport", ">= 3.2.0", "~> 6.1.0"
```

Comment:
Dropping support for Rails 3.2 is probably necessary, but also constitutes a major version bump. Dropping it while adding Rails 7.0 support seems like a good time to do it.

Looks like I flubbed the version bump when aligning this with Rails versions, so I'll also need to iron that out.